### ISSUE TITLE
Improvements/fixes of the Tribler statistics page

### ISSUE THREAD

**Issue by devos50** [Tribler/tribler#5387]
We display a few network statistics on a [seperate statistics website](http://statistics.tribler.org). However, this website is not actively maintained and has various errors.
- The site is currently served from one of our own servers. It can be served from Github when we move the code to a separate repository.
- The 'user distribution' graph is not working.
- It seems that the server that is performing the network walk is under high load. It has been running for almost 2 years without a restart.

**Comment by hbiyik** [comment]
I would really like to see the jump when kodi addon is released, I am expecting 1 million unique users daily when addon is poppulated, so an indicator of the exit node health would be great, it may literally choke the network.

But i think there is not way to get health status (bandwith allocated and free etc) without modifying the tunnel community right?

**Comment by devos50** [comment]
> I would really like to see the jump when kodi addon is released, I am expecting 1 million unique users daily when addon is poppulated

Yes, this would be great to see (also good for marketing purposes)!

> But i think there is not way to get health status (bandwith allocated and free etc) without modifying the tunnel community right?

Our current statistics page only shows the number of online users and their estimated geographical location. Eventually, we want to move to a single dashboard (see #4999) where we display all kind of network statistics. Getting and displaying statistics from our anonymous overlay is tricky since that might compromise anonymity.

**Comment by qstokkink** [comment]
Our statistics work again but they are served from https://release.tribler.org/dashboard/

The tribler.org website still points to http://statistics.tribler.org/ though, under `Tribler/Statistics`.
